src/Sharpliner/AzureDevOps/Parameter.cs

```diff
@@ -40,6 +41,24 @@ public Parameter(string name, string? displayName = null)
         Name = name;
         DisplayName = displayName;
     }
+
+    /// <summary>
+    /// Converts a <see cref="Parameter"/> to a <see cref="string"/> representation of the reference to the parameter.
+    /// </summary>
+    /// <param name="parameter">The parameter.</param>
+    public static implicit operator string(Parameter parameter) => new ParameterReference(parameter.Name);
+
+    /// <summary>
+    /// Converts a <see cref="Parameter"/> to a <see cref="IfExpression"/> by getting the reference to the parameter.
+    /// </summary>
+    /// <param name="parameter">The parameter.</param>
+    public static implicit operator IfExpression(Parameter parameter) => new ParameterReference(parameter.Name);
+
+    /// <summary>
+    /// Converts a <see cref="Parameter"/> to a <see cref="InlineExpression"/> by getting the reference to the parameter.
+    /// </summary>
+    /// <param name="parameter">The parameter.</param>
+    public static implicit operator InlineExpression(Parameter parameter) => new ParameterReference(parameter.Name);
```

Comment:
I think we need all 3 overloads since an operator for `ParameterReference` won't be enough for an implicit operator (but converting to these different types by leveraging the implicit operator of `ParameterReference` does the job here 😀)